### ISSUE TITLE
Homepage actions button on / index page

### ISSUE THREAD

**Issue by simonw** [simonw/datasette#2298]
The blue table/database/query action buttons are great, I want one for the homepage (at `/`) as well. Could be used for features like:

- Customize homepage layout
- Edit metadata

I didn't consider one of these before because of the `menu_links()` hook, but that affects the burger menu that is shown on every page. Having an actions menu for options that only really make sense in the context of the index page makes sense to me.

**Comment by simonw** [comment]
I thought about calling it "Instance actions" but I decided "Homepage actions" would be more obvious.

"Homepage" or "Home page"? I like one word because it is consistent with how "Database actions" and "Table actions" and "Query actions" are also one word.

I checked AP styleguide and while they don't have anything to say about "homepage" directly their own site does use that term consistently itself: https://www.apstylebook.com/help?query=homepage

![Screenshot of a search for homepage on apstylebook.com](https://github.com/simonw/datasette/assets/9599/fe79a8f3-a834-46a4-a931-37333efee1c7)
